lib/md040.js

```diff
@@ -2,16 +2,27 @@
 
 "use strict";
 
-const { addErrorContext, filterTokens } = require("../helpers");
+const { addErrorContext, filterTokens, addError } = require("../helpers");
 
 module.exports = {
   "names": [ "MD040", "fenced-code-language" ],
   "description": "Fenced code blocks should have a language specified",
   "tags": [ "code", "language" ],
   "function": function MD040(params, onError) {
+    let allowList = params.config.allow_list;
+    allowList = Array.isArray(allowList) ? allowList : null;
     filterTokens(params, "fence", function forToken(token) {
-      if (!token.info.trim()) {
+      const lang = token.info.trim();
+      if (lang === "") {
         addErrorContext(onError, token.lineNumber, token.line);
+        return;
```

Comment:
Replace with else.

Reply:
Eslint won't let me use `else`:


<img width="745" alt="" src="https://user-images.githubusercontent.com/1091472/196066790-a1d97141-2e2f-48ec-8f2b-7f4cafb6251e.png">

Should I disable the rule?

Reply:
`if (lang) {...} else if (allowed && includes) {...}` should be allowed.